### DISCUSSION TITLE
Right-justify panel buttons on header

### DISCUSSION THREAD

**Discussion by takaji10** [TT-ReBORN/Georgia-ReBORN#22] [Q&A] [answered]
I am frequently confusing `Options` and `Details` in the top bar. It would make sense to right-justify `Details`, `Library`, `Biography`, `Lyrics`, and `Rating`, as these are buttons and not menu items (with the exception of `Rating`).

Current layout:
![image](https://user-images.githubusercontent.com/1534470/125470446-6aef62e3-defd-4deb-acd9-d99cc496c88b.png)

Proposed layout:
![image](https://user-images.githubusercontent.com/1534470/125470723-30d9a57f-1da5-4479-9dc9-4b73f4077d3a.png)

Also consider applying a rounded border around each button in its default state, and changing the button to a dark/shaded version when the current panel is selected to further indicate to the user that these are buttons and not menu items.

**Comment by TT-ReBORN** [comment, marked as answer]
No, I won't do that because you should change the player size before proposing this idea.

Player Size 'Small' on FHD resolution is the only player size that has this linked compact menu and is the best design choice. 
Player Size Normal, Big + Fullscreen in FHD & Player Size 'Small' + 'Normal' + 'Big' + Fullscreen in 4K,
the menu is centered and it will stay that way.

Btw, you can always make a fork and customize how you personally want it.

**Reply by takaji10** [reply]
Okay, this makes sense. I didn't try ever making my player fullscreen, and I can see now how it works. It's not an issue when the player is larger, it's only confusing on the "small" player view.

Perhaps it would still make sense to draw a rounded border around these buttons, and shade them when they're clicked and currently displaying in the player?

Edit: Is it not possible to remove the small/medium/large options and just have the top bar dynamically change as the user increases the size of the window/uses fullscreen? I just switched my player size to "Medium" and resized it back to be 1/4 of my display, and now the buttons are centred.

**Reply by TT-ReBORN** [reply]
No, not gonna happen. The player sizes will stay as they are, they have a purpose for layout modes and 4K resolution.

Also I don't like to shade/fill the active buttons, they are already indicated when active through darker font color and
button outline. This design choice makes it look light and clean, shading/filling buttons is distracting on the overall design.

**Comment by TT-ReBORN** [comment]
Also another important thing about the player sizes is that they are predefined sizes that makes pixel perfect player sizes that matches artwork cover size with playlist height. You can't do that by manually dragging the player size, it always needs to be painstakingly adjusted.

These are limitations with foo_ui_hacks and SMP, no proportional aspect ratio resize possible...

P.S Player Size 'Small' + 'Normal' + 'Big' are not just 3 sizes, they are actually 12 player sizes ( see Github front page ).
FHD 3 for default mode, 3 for playlist mode
4K 3 for default mode, 3 for playlist mode

**Reply by takaji10** [reply]
Thanks for explaining it to me. I'm trying to work my way through all the available options with this layout and understand everything, and I guess this one slipped by me!

I just tried the 4K mode and it's pretty cool - maybe I'll use it when I'm hosting a party? :)

**Reply by TT-ReBORN** [reply]
Yes totally! 
The player looks best in 4K resolution, everything looks more crisp, not just the fonts... =)
Once 4K, never go back!
